### ISSUE TITLE
Conversation Settings menu is not accessible via keyboard

### ISSUE THREAD

**Issue by krallowski** [humhub/mail#502]
### Description
The settings menu on the top right in the conversation element cannot be accessed via keyboard because it is not part of the tab order.

### Steps to reproduce
1. Open a conversation
2. Press tab to tab through interactive elements and try to reach the dropdown arrow

### Expected behavior
At some point the arrow gets focus and I can press enter to open the menu

### Current behavior
Tabbing skips the element

### Possible Solution
Add a button element to the icon: https://github.com/humhub/mail/blob/7540f91c2e764ede2b24d8ff191b4c9bf312164c/widgets/views/conversationSettingsMenu.php#L26

### Additional info

Appeared on Firefox, HumHub version 1.18
